plugins/outputs/cloudwatchlogs/pusher.go

```diff
@@ -55,6 +55,7 @@ type pusher struct {
 
 	initNonBlockingChOnce sync.Once
 	startNonBlockCh       chan struct{}
+	wg                    sync.WaitGroup
```

Comment:
Can we follow the structure that already exists in the `cloudwatch` output plugin where this attribute is a pointer to a WaitGroup?
```suggestion
	wg                    *sync.WaitGroup
```